content/en/load-testing/11-glossary.md

```diff
@@ -87,10 +87,9 @@ Main problems with pure recored test scripts are:
     structured test suite. Missing reusability and difficult maintenance
 -   Often insufficient validation of results
 
-The XLT Script Developer is an advanced Capture & Replay Tool that
-features a broad range of additional functionality and is integrated
-into the large XLT testing framework and therefore is able to overcome
-the disadvantages named above.
+The XLT Script Developer was such an advanced Capture & Replay Tool, 
```

Comment:
sadly ->  but it was discounted when Mozilla removed the XUL-application API from Firefox.